lumen/ai/models.py

```diff
@@ -170,35 +170,45 @@ class LineChange(BaseModel):
         To remove a line set this to an empty string."""
     )
 
+class BaseStep(BaseModel):
```

Comment:
Should this be called PlanStep instead? Or what's the "Base" referencing (will there be subclasses for this)?